idea/src/org/jetbrains/kotlin/idea/codeInsight/surroundWith/expression/KotlinControlFlowExpressionSurrounderBase.kt

```diff
@@ -29,6 +29,7 @@ abstract class KotlinControlFlowExpressionSurrounderBase : KotlinExpressionSurro
 
     override fun isApplicable(expression: KtExpression) = true
     override fun isApplicableToUnit() = true
+    override fun isApplicableToStatements() = false
```

Comment:
It looks like `isApplicableToStatements() == !isApplicableToUnit()` is always true while there's separation to to `KotlinExpressionSurrounder` and `KotlinStatementsSurrounder`.